components/d2l-filter-dropdown/d2l-filter-dropdown-option.js

```diff
@@ -0,0 +1,49 @@
+import '@polymer/polymer/polymer-legacy.js';
+
+import 'd2l-icons/d2l-icons.js';
+import 'd2l-menu/d2l-menu-item-selectable-styles.js';
+import 'd2l-menu/d2l-menu-item-selectable-behavior.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<dom-module id="d2l-filter-dropdown-option">
+	<template strip-whitespace="">
+		<style include="d2l-menu-item-selectable-styles">
+			:host > span {
+				white-space: normal;
+			}
+		</style>
+		<d2l-icon icon="d2l-tier1:check" aria-hidden="true"></d2l-icon>
```

Comment:
I am surprised the `aria-hidden` is needed here. (?)

Reply:
Discussed offline - going to leave for now in case there's an accessibility reason, but we should revisit whether this is really needed when we convert to Lit.  FYI @margaree